xpad.c

```diff
@@ -1379,7 +1388,9 @@ static void xpad_set_up_abs(struct input_dev *input_dev, signed short abs)
 		break;
 	case ABS_Z:
 	case ABS_RZ:	/* the triggers (if mapped to axes) */
-		if (xpad->xtype == XTYPE_XBOXONE)
+		if (xpad->mapping & MAP_WHEEL)
```

Comment:
I think my preference would be:

wheel - ABS_X (-32768, center at 0, 32768)
brake - ABS_Z (0 to 65535)
gas - ABS_RZ (0 to 65535)